test/parallel_exec.jl

```diff
@@ -477,29 +491,6 @@ test_channel(RemoteChannel(()->Channel(10)))
 c=Channel{Int}(1)
 @test_throws MethodError put!(c, "Hello")
 
-c=Channel(256)
-# Test growth of channel
-@test c.szp1 <= 33
-for x in 1:40
-  put!(c, x)
-end
-@test c.szp1 <= 65
-for x in 1:39
-  take!(c)
-end
-for x in 1:64
-  put!(c, x)
-end
-@test (c.szp1 > 65) && (c.szp1 <= 129)
-for x in 1:39
-  take!(c)
-end
-@test fetch(c) == 39
-for x in 1:26
-  take!(c)
-end
-@test isready(c) == false
```

Comment:
is this set of tests really exactly redundant with the above? that's not immediately clear to me if so